DiscordChatExporter.Cli/Commands/Base/ExportCommandBase.cs

```diff
@@ -69,6 +69,34 @@ protected async ValueTask ExportAsync(IConsole console, Guild guild, Channel cha
             console.Output.WriteLine("Done.");
         }
 
+        public void ExecuteAsync()
+        {
+            if (ShouldReuseMedia && !ShouldDownloadMedia)
+            {
+                var shouldDownloadMediaOptionName = GetAttributeName(nameof(ShouldDownloadMedia));
+                var shouldReuseMediaOptionName = GetAttributeName(nameof(ShouldReuseMedia));
+
+                Console.WriteLine($"The \"--{shouldReuseMediaOptionName}\" option cannot be used without the \"--{shouldDownloadMediaOptionName}\" option.");
+                Environment.Exit(1);
```

Comment:
This can be replaced with:

```suggestion
                throw new CommandException($"The --reuse-media option cannot be used without the --media option.");
```

Reply:
Oh, I thought throwing an exception would result in an ugly console message, but apparently it doesn't. Neat!

Reply:
It was made specifically for this :)